README.md

```diff
@@ -184,7 +184,7 @@ We provide a training script using the launch utility provided by PyTorch.
 Supported arguments are:
 
 - --validate: perform evaluation every k (default=1) epochs during the training.
-- --work_dir <WORK_DIR>: if specified, the path in config file will be overwritten.
+- --work_dir <WORK_DIR>: if not specified, the path in config file will be overwritten.
```

Comment:
I think this is not a typo. If `--work_dir` is not specified, the path in config file will be used, otherwise the specified work_dir will be used.

Reply:
Oh, I wrongly take the subject of `specified` to be `work_dir`

Reply:
Oh, the word `overwritten` is confusing. I think it means that the `path` will be overwritten, discarding any content in it. Actually, it means the `value` in `python` will be overwritten with a new `value`.